### ISSUE TITLE
Do we have bruce's full 2 part intro to raku?

### ISSUE THREAD

**Issue by toddr** [perlconference/tprc-2023-tor#8]

**Comment by toddr** [comment]
This is only the first 2 hours? https://www.youtube.com/watch?v=eb-j1rxs7sc

**Comment by toddr** [comment]
https://tprc2023.sched.com/grid/#2023-07-11

**Comment by jeturcotte** [comment]
Yes, on the spreadsheet (https://docs.google.com/spreadsheets/d/16Yw1OIKvDLcIfJs4ky22NJ8d0k1kdtibQrBhHB378NY/edit?usp=sharing), it indicates that the beginning is missing and that I need from him some audio to bridge the loss... or his designating a place a little further in that feels like a less-confusing place to start.

Mentioned this to bruce over slack directly as well as to a couple different people who asked after it; I haven't heard back.

**Comment by toddr** [comment]
@Util halp?

**Comment by Util** [comment]
@jeturcotte The "missing 4 minutes" talk you are referring to is the Command-line Filters talk. I have opened #12 to track that separate issue, and replied to your Slack message about it. 

The talk in *this* #8 issue is "Raku for Beginners", which was missing the whole 1.5 hour second session. First session was `D1K2` (with the screen-cap weirdly split into 1h45min and 15min), second was `D1K3`. People (including me) were assuming `D1K3` would be uploaded later to Youtube as a separate video. 

In the `progress` spreadsheet, session `D1K2` says:

    missing footage found; added; recompiled, uploading now

I am unsure if "missing footage" referred to

    1) the weird split screen-cap for D1K2, and so "uploading now" resulted in the 2-hour video currently on Youtube, or
    2) the `D1K3` files that no one listed on the spreadsheet, and are planning on replacing the already uploaded file with a full 3h30min integrated version.

If (1), then (I think) the `D1K3` files need to be processed. I have them on the hard drive; just let me know if they were not sent to you.
If (2), then perhaps the upload failed, or the new 3h30min version is not public?
